data/generateData.js

```diff
@@ -75,10 +76,15 @@ function getStatsByPlayer(acc, { players }, index, rounds) {
       scoreDist[score] = idx(scoreDist, score, 0) + 1;
 
       const cumHoleScore = idx(scoresByHole[hole], 'cumHoleScore', 0) + score;
+      const avgHoleScore = getAvg(cumHoleScore, roundsPlayed);
+      const recentAvgHoleScore = roundsPlayed > RECENT_ROUND_COUNT ?
+        idx(scoresByHole[hole], 'recentAvgHoleScore', 0) :
```

Comment:
I cannot figure out how the value for `recentAvgHoleScore` gets populated. I guess I'll merge it and play around with it.

Reply:
It's the oddity of the accumulator stuff. ScoresByHole is taken from the `acc`, which is basically the "result so far". So the `idx` is saying, ~"try to get this value from the result so far. if it's not there already, use this default value".

Reply:
Okay, I get it now. The scores are sorted in *reverse* chronological order (i.e., most recent first). So, starting from the 9th round on (when `roundsPlayed > RECENT_ROUND_COUNT`), `'recentAvgHoleScore'` never changes.